src/chunk.c

```diff
@@ -2580,15 +2594,34 @@ ts_chunk_drop_chunks(PG_FUNCTION_ARGS)
 				 errmsg("older_than and newer_than timestamps provided to drop_chunks cannot both "
 						"be NULL")));
 
+	if (schema_name == NULL)
+	{
+		/* default schema is public */
+		schema_name = (NameData *) palloc0(NAMEDATALEN);
+		namestrcpy(schema_name, "public");
+	}
 	ht_oids = ts_hypertable_get_all_by_name(schema_name, table_name, CurrentMemoryContext);
 
 	if (table_name != NULL)
 	{
+		ContinuousAgg *ca = NULL;
 		if (ht_oids == NIL)
-			ereport(ERROR,
-					(errcode(ERRCODE_TS_HYPERTABLE_NOT_EXIST),
-					 errmsg("hypertable \"%s\" does not exist", NameStr(*table_name))));
+		{
+			ca = ts_continuous_agg_find_by_view_name(NameStr(*schema_name), NameStr(*table_name));
+			if (ca == NULL)
+				ereport(ERROR,
+						(errcode(ERRCODE_TS_HYPERTABLE_NOT_EXIST),
+						 errmsg("hypertable \"%s\" does not exist", NameStr(*table_name))));
```

Comment:
I guess this error is a bit confusing in case one passed in the view name. I suggest updating the error to say something like: `errmsg("%s is not a hypertable or continuous aggregate view")` `errhint("It is only possible to drop chunks on a user-created hypertable or continuous aggregate view.")`